dev-util/android-sdk-update-manager/android-sdk-update-manager-26.1.1.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+PYTHON_COMPAT=(python3_{10..11})
+
+inherit python-r1 udev distutils-r1
+
+DESCRIPTION="Open Handset Alliance's Android SDK"
+HOMEPAGE="https://developer.android.com"
+SRC_URI="https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip"
+RESTRICT="mirror"
+
+LICENSE="android"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="app-arch/tar
+		app-arch/unzip
+		app-arch/gzip
+		acct-group/android"
+
+RDEPEND=">=virtual/jdk-1.5
+	>=dev-java/ant-core-1.6.5
+	dev-java/swt:3.7[cairo]
+	sys-libs/ncurses-compat:5[abi_x86_32(-)]
+	sys-libs/zlib[abi_x86_32(-)]
+"
+
+ANDROID_SDK_DIR="/opt/${PN}"
+QA_FLAGS_IGNORED_x86="
+	${ANDROID_SDK_DIR/\//}/tools/adb
+	${ANDROID_SDK_DIR/\//}/tools/mksdcard
+	${ANDROID_SDK_DIR/\//}/tools/sqlite3
+	${ANDROID_SDK_DIR/\//}/tools/hprof-conv
+	${ANDROID_SDK_DIR/\//}/tools/zipalign
+	${ANDROID_SDK_DIR/\//}/tools/dmtracedump
+"
+QA_FLAGS_IGNORED_amd64="${QA_FLAGS_IGNORED_x86}"
```

Comment:
I've never seen using it in this way.. Isn't enough to set 

```
QA_FLAGS_IGNORED="
	${ANDROID_SDK_DIR/\//}/tools/adb
	${ANDROID_SDK_DIR/\//}/tools/mksdcard
	${ANDROID_SDK_DIR/\//}/tools/sqlite3
	${ANDROID_SDK_DIR/\//}/tools/hprof-conv
	${ANDROID_SDK_DIR/\//}/tools/zipalign
	${ANDROID_SDK_DIR/\//}/tools/dmtracedump
"
```

as they are the same? My couriosity

Reply:
Also, if you prefer, you can just use that.
`
QA_FLAGS_IGNORED="${ANDROID_SDK_DIR/\//}/tools/{adb,mksdcard,sqlite3,hprof-conv,zipalign,dmtracedump}"`